docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.7"
```

Comment:
docker-compose is a good sample to bootstrap a docker. But I personally do not want it to be here since people will take it "it should be maintained" for granted. 
I prefer to provide lines in documents in style as `docker run ....`